docs/content/blog/cncf.md

```diff
@@ -2,7 +2,34 @@
 title: "Porter Has Joined the CNCF"
 description: "Porter has been accepted into the Cloud Native Computing Foundation (CNCF)"
 date: "2020-09-24"
-draft: true
 ---
 
-Porter has been accepted into the Cloud Native Computing Foundation (CNCF)...
+You may have noticed a flurry of activity recently on the Porter repositories
+and wondered what was afoot. It was all in preparation for the big reveal:
+Porter has been accepted into the Cloud Native Computing Foundation (CNCF)!<!--more-->
+
+When Jeremy Rickard and I created Porter less than two years ago, this is what
+we hoped for. We wanted to create a user-friendly tool for that would make Cloud
+Native Application Bundles (CNAB) accessible and deployments less stressful. It
+had to be user-first and community-first, without vendors getting in the way. So
+our sights were set on finding an independent home at the CNCF from day one. It
+means a lot to us to have built a community around Porter and tricked the CNCF
+into letting us in. 😉
+
```

Comment:
When Jeremy Rickard and I created Porter less than two years ago, we wanted to create a community-led project that would encourage collaboration across vendors to make Cloud Native Application Bundles (CNAB) accessible and maintainable. In joining the CNCF, we can continue to support that mission of vendor neutral independence while growing the community of practitioners who can use and contribute to Porter's development.

Reply:
Today, the Porter Maintainers are happy to announce that Porter is now an official CNCF Sandbox project. By contributing Porter to the CNCF, we hope to encourage the adoption of Porter to ease the creation and use of bundles and to see more companies contribute mixins.